types/bwip-js/bwip-js-tests.ts

```diff
@@ -37,3 +37,16 @@ bwipjs.toBuffer({
         // png.readUInt32BE(20) : PNG image height
     }
 });
+
+// Browser canvas implementation
+const canvas = document.createElement('canvas') as HTMLCanvasElement;
+bwipjs(canvas, {
+    bcid: 'qrcode',
+    text: 'example',
+}, (err, cvs) => {
+    if (err) {
```

Comment:
Add `$ExpectType` comment to verify type instead of using console.log

Reply:
Should I also change the existing tests?  (I was just copying them)